app/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundService.scala

```diff
@@ -128,7 +128,10 @@ class OutboundService @Inject() (
   private def processSendingResult(message: MessageRequest, soapRequest: SoapRequest, httpStatus: Int): OutboundSoapMessage = {
     val globalId: UUID = randomUUID
     val messageId      = message.addressing.messageId
-
+    val privHeaders = message.privateHeaders match {
+      case Some(headers) => headers
+      case None => List()
```

Comment:
As per previous remark: if we have no private headers then why add an empty list to the message?

Reply:
> As per previous remark: if we have no private headers then why add an empty list to the message?

This is the requirement as I said above